qa/acceptance/spec/centos/spec_helper.rb

```diff
@@ -0,0 +1,20 @@
+require 'runner-tool'
+require_relative '../../../rspec/helpers'
+require_relative '../../../rspec/matchers'
+require_relative '../config_helper'
+
+def vagrant_boxes
+  [
+    "centos-6",
+    "centos-7"
+  ]
+end
```

Comment:
If we use a hash for settings up the vagrant boxes we would be able to reuse the data here.
see my comment on https://github.com/elastic/logstash/pull/5138/files#r60282516